### ISSUE TITLE
Abstract method verbose in FluxCalibration 

### ISSUE THREAD

**Issue by miguelglezb** [pykoala/pykoala#453]
When running `reduction_sequence_science.ipynb`, the second to last cell is failing:

`TypeError: Can't instantiate abstract class FluxCalibration with abstract method verbose`

I tracked the error to the CorrectionsBase class, particularly in the verbose decorators:

```python
    @property
    @abstractmethod
    def verbose(self):
        return self._verbose

    @verbose.setter
    def verbose(self, verbose):
        self._verbose = verbose
```

I tried to fix it by commenting the `@asbractmethod` but did not work. Also, that may break others instances. The other option that I'm trying is to redefine the function `list_available_stars(verbose=True)` or change the usage of attribute verbose for this class.

**Comment by miguelglezb** [comment]
Issue has been fixed in PR #456, please review.

**Comment by miguelglezb** [comment]
This issue has been fixed and tests were passed. I'm closing this issue now.
